website/docs/Ganglion/09-Ganglion_Programming_Tutorial.md

```diff
@@ -2,41 +2,45 @@
+This guide will walk you through how to update your Ganglion firmware. Downloading the latest binary and Over The Air programming (OTA) makes updating the Ganglion a breeze with a mobile device. If you want to compile the code in Arduino, continue to the [Building From Source](#ganglion-programming-tutorial-building-from-source) portion of the guide.
 
 ## Download The Latest Builds
 
 - [OpenBCI GUI](https://github.com/OpenBCI/OpenBCI_GUI/releases)  
-- [Ganglion Firmware v3.0.0](https://github.com/OpenBCI/OpenBCI_Ganglion_Library/releases/download/v3.0.0/DefaultGanglion3.0.1.zip)
+- [Ganglion Firmware v3.0.0](https://github.com/OpenBCI/OpenBCI_Ganglion_Library/releases/download/v3.0.0/DefaultGanglion3.0.0.zip)
 
-If you prefer to use the precompiled binaries, jump down to [setup your mobile device for OTA programming](#ganglion-programming-tutorial-setup-mobile-device-for-ota-programming) to continue.
+If you are planning to program your Ganglion using the precompiled binaries, jump down to [Setup Mobile Device For OTA Programming](#setup-mobile-device-for-ota-programming) section to continue.
 
 ## Building From Source
 
-The Simblee (website discontinued) radio module that is at the heart of the Ganglion board offers the ability to reprogram Over The Air `OTA`. The method for doing this in a nutshell is:  
+The Simblee (website discontinued) radio module that is at the heart of the Ganglion board offers the ability to reprogram Over The Air (OTA). The method for doing this in a nutshell is:  
 
-1.  Create new Ganglion firmware, or modify existing code in Arduino IDE
-2.  Generate a compiled HEX file in the Arduino IDE
-3.  Compress it into a ZIP with some other important files
-4.  Transfer the ZIP file to your phone or tablet
+1.  Create new Ganglion firmware, or modify the existing code in the Arduino IDE
+2.  Generate a compiled binary in the Arduino IDE
+3.  Compress it into a `.zip` archive with some other important files
+4.  Transfer the `.zip` archive to your phone or tablet
 5.  Connect your phone or tablet to the Ganglion
-6.  Upload new code Over The Air
+6.  Upload new code
 
-The following tutorials will get your computer and the Arduino IDE set up to create the correct ZIP file, and also show how to use your phone or tablet to upload new code to the Ganglion. This guide will also teach you how to upload code to your ganglion for the first time.
+The following instructions will get your computer and the Arduino IDE set up to create the correct `.zip` archive and show how to use your phone or tablet to upload new code to the Ganglion.
 
-**_IMPORTANT: Ganglions shipped prior to February 27, 2017 will NOT program Over The Air for the first time! If you purchased your Ganglion prior to 2/27/17, Please follow the instructions below up till How to create an OTA File and read the instructions the bottom of this page to program your Ganglion for the first time._**
+:::caution
```

Comment:
I was going to put this  as `important` admonition.

Reply:
You can also still do bold + italics inside admonition if you want to pull out all the stops.